tests/functional/diff/test_diff.py

```diff
@@ -561,3 +561,38 @@ def test_local_compare_res_count_unresovled(self):
         self.assertEqual(count, 5)
         count = len(re.findall(r'\[unix\.Malloc\]', out))
         self.assertEqual(count, 1)
+
+    def test_local_compare_res_count_unresovled_regex(self):
+        """
+        Count the unresolved results with no filter in local compare mode.
```

Comment:
A test case with a filter would be nice too.

Reply:
There already is a test case for filtering above. This test case checks only the regex in baseline name.